src/richie/plugins/simple_picture/models.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+"""
+Enables the user to add an "Image" plugin that displays an image
+using the HTML <img> tag.
+"""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from cms.models import CMSPlugin
+from filer.fields.image import FilerImageField
+
+from .defaults import SIMPLEPICTURE_PRESETS
+
+# pylint: disable=no-member
+
+
+class SimplePicture(CMSPlugin):
+    """
+    Renders an image with the option of adding a link
+    """
+
+    picture = FilerImageField(
+        verbose_name=_("Image"), on_delete=models.PROTECT, related_name="picture"
+    )
+
+    def __str__(self):
+        return self.picture.label
+
+    def get_info(self, preset_name, alt=None):
+        """
+        Compute picture information for a given preset defined in settings.
+
+        A preset is of the form:
+
+        "default": {
+            # easythumbnail options
+            "src": {"size": (1000, 1000), "crop": "smart"},
+            # list of tuples ({easythumbnail options}, {srcset description})
```

Comment:
Idem: which tuples?

Reply:
Fixed. This was a comment left from an earlier implementation.